test/WebJobs.Script.Tests/Description/Proxy/ProxyFunctionDescriptorProviderTests.cs

```diff
@@ -42,6 +42,7 @@ public ProxyFunctionDescriptorProviderTests()
 
             _proxyClient = GetMockProxyClient();
             _settingsManager = ScriptSettingsManager.Instance;
+            _settingsManager.Reset();
```

Comment:
? Tests flaking out or something?

Reply:
Yeah. Fabio recommended that as a fix. I guess we're planning on changing how it works later, anyway?

Reply:
Yes. This is temporary and caused by other tests essentially setting values that remain cached on the same instance. We'll be removing this type soon.